x-pack/lib/config_management/bootstrap_check.rb

```diff
@@ -14,7 +14,7 @@ module ConfigManagement
     class BootstrapCheck
       include LogStash::Util::Loggable
 
-      # pipeline ID must begin with a letter or underscore and contain only letters, underscores, dashes, and numbers
+      # pipeline ID must begin with a letter or underscore and contain only letters, underscores, dashes, hyphens and numbers
```

Comment:
Let's keep the Oxford comma for clarity please

```suggestion
      # pipeline ID must begin with a letter or underscore and contain only letters, underscores, dashes, hyphens, and numbers
```